src/common/components/textarea-autocomplete/index.tsx

```diff
@@ -79,39 +81,32 @@ export default class TextareaAutocomplete extends BaseComponent<any, State> {
 				onChange={this.handleChange}
 				minChar={2}
 				trigger={{
-					"@": {
+					["@"]: {
 						dataProvider: token => {
 							clearTimeout(timer);
-							if (token.indexOf('/')===-1) {
-								return new Promise((resolve) => {
-									timer = setTimeout(async () => {
+							return new Promise((resolve) => {
+								timer = setTimeout(async () => {
+									if(token.includes("/")){
+										searchPath(activeUser, token).then(resp => {
+											resolve(resp)
+										})}
+									else {
 										let suggestions = await lookupAccounts(token, 5)
 										resolve(suggestions)
-									}, 300);
-								});
-							} else {
-								return [];
-								/*new Promise((resolve) => {
-									const splt = token.split('/');
-									if (splt[1]) {
-										timer = setTimeout(async () => {
-											let suggestions = await searchPath(this.props.activeUser.username, token);
-											resolve(suggestions)
-										}, 1000);
 									}
-								});*/
-							}
+								}, 300);
+							});
 						},
 						component: (props: any) => {
 							return (
 								<>
-									{props.entity.indexOf('/')===-1 && UserAvatar({ global: this.props.global, username: props.entity, size: "small" })}
+									{UserAvatar({ global: this.props.global, username: props.entity, size: "small" })}
```

Comment:
There is no avatar when selecting path, so there should be check if search is user or path. @mghazanfar